<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EasyPBI Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_pbi">
      <attribute name="title">
       <string>PBI Settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>PBI Output Dir</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="line_pbioutdir">
           <property name="toolTip">
            <string>Local directory for built PBI's to be placed.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="push_pbioutdir">
           <property name="toolTip">
            <string>Browse local system to select a PBI output directory</string>
           </property>
           <property name="text">
            <string>Select</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="check_signpbi">
         <property name="toolTip">
          <string>Digitally sign any PBI's that are created for tamper-evident distribution</string>
         </property>
         <property name="text">
          <string>Digitally Sign PBI</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_sig">
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Signature File</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="line_sigfile">
            <property name="toolTip">
             <string>Local file that contains the openssl key</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="push_sigfile">
            <property name="toolTip">
             <string>Browse for the openssl key file</string>
            </property>
            <property name="text">
             <string>Select</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="check_usetmpfs">
         <property name="toolTip">
          <string>Use system memory to speed up PBI build processes.</string>
         </property>
         <property name="text">
          <string>Use TMPFS</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="check_usepkgcache">
         <property name="toolTip">
          <string>Greatly speeds up the PBI build process by preventing re-compilation of previously built ports.</string>
         </property>
         <property name="text">
          <string>Use Package Caching</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_pkgcache">
         <property name="title">
          <string>Package Caching Options</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Packages to Ignore</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="list_badpkg">
              <property name="toolTip">
               <string>Listed packages will always be ignored during PBI builds.</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QPushButton" name="push_addpkg">
                <property name="toolTip">
                 <string>Add a currently cached package to be ignored.</string>
                </property>
                <property name="text">
                 <string>Add</string>
                </property>
                <property name="icon">
                 <iconset resource="EasyPBI.qrc">
                  <normaloff>:/png/icons/plus.png</normaloff>:/png/icons/plus.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="push_rmpkg">
                <property name="toolTip">
                 <string>Remove the selected package from the ignore list.</string>
                </property>
                <property name="text">
                 <string>Remove</string>
                </property>
                <property name="icon">
                 <iconset resource="EasyPBI.qrc">
                  <normaloff>:/png/icons/minus.png</normaloff>:/png/icons/minus.png</iconset>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QPushButton" name="push_clearcache">
              <property name="toolTip">
               <string>Delete all packages currently in the cache.</string>
              </property>
              <property name="text">
               <string>Clear Cache</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_paths">
      <attribute name="title">
       <string>Local Paths</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::ExpandingFieldsGrow</enum>
       </property>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Module Dir</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="line_moduledir">
           <property name="toolTip">
            <string>Default directory for all PBI modules to be created</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="push_moduledir">
           <property name="toolTip">
            <string>Select a local directory for PBI modules.</string>
           </property>
           <property name="text">
            <string>Select</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Icon Dir</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="line_icondir">
           <property name="toolTip">
            <string>Default directory to open when adding icons/files to a module</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="push_icondir">
           <property name="toolTip">
            <string>Select a local directory.</string>
           </property>
           <property name="text">
            <string>Select</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Icon FIle</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLineEdit" name="line_defaulticon">
           <property name="toolTip">
            <string>Default PNG icon to use for PBI modules.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="push_defaulticon">
           <property name="toolTip">
            <string>Browse local system for a default PNG icon.</string>
           </property>
           <property name="text">
            <string>Select</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_utils">
      <attribute name="title">
       <string>Detected Utilities</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="push_resetutils">
         <property name="toolTip">
          <string>Have EasyPBI re-scan the system for available utilities.</string>
         </property>
         <property name="text">
          <string>Auto-Detect</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>pbi_makeport</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLineEdit" name="line_makeport">
           <property name="toolTip">
            <string>Full path to the &quot;pbi_makeport&quot; binary</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="push_makeport">
           <property name="toolTip">
            <string>Browse system for the local binary.</string>
           </property>
           <property name="text">
            <string>Select</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>pbi_create</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLineEdit" name="line_pbicreate">
           <property name="toolTip">
            <string>Full path to the &quot;pbi_create&quot; binary</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="push_pbicreate">
           <property name="toolTip">
            <string>Browse system for the local binary.</string>
           </property>
           <property name="text">
            <string>Select</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Switch User Utility</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLineEdit" name="line_suutility">
           <property name="toolTip">
            <string>Full path to the graphical &quot;switch user&quot; utility (pc-su, gksu, kdesu, qsu, etc..)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="push_suutility">
           <property name="toolTip">
            <string>Browse system for the local binary.</string>
           </property>
           <property name="text">
            <string>Select</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>FreeBSD Ports Dir</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="line_portsdir">
           <property name="toolTip">
            <string>Location of the FreeBSD ports tree on the local system.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="push_portsdir">
           <property name="toolTip">
            <string>Select the FreeBSD ports tree directory.</string>
           </property>
           <property name="text">
            <string>Select</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="EasyPBI.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
